tasks/insights-client.yml

```
@@ -36,6 +36,17 @@
     insertafter: "#auto_update"
     line: auto_update={{ rhc_insights.autoupdate | d(true) | bool }}
 
+- name: Configure ansible host
+  lineinfile:
+    path: "{{ __rhc_insights_conf }}"
+    regexp: "^ansible_host"
+    insertafter: "#ansible_host"
+    line: ansible_host={{ rhc_insights.ansible_host }}
+  register: __insights_ansible_host_configured
+  when:
```

```suggestion
  when:
    - rhc_insights.ansible_host is defined
```
afaict it is possible for `rhc_insights.ansible_host` to be undefined, and that would cause an error

---

hm how so? since it's in the defaults, it should be always defined, shouldn't it? to me this looks like a mistake I did in the past, see #127

---

++ for Pino’s „how so?“